backend/apps/slack/admin/bot_interaction.py

```
@@ -0,0 +1,80 @@
+        "user_message",
+        "intent_category",
+    )
+ 
+    @admin.display(description="Message")
+    def short_message(self, obj):
+        """Truncated user message for list display."""
+        return obj.user_message[:60] + "…" if len(obj.user_message) > 60 else obj.user_message
+ 
+ 
+admin.site.register(BotInteraction, BotInteractionAdmin)
+ 
```

Ruff (0.15.18)

[warning] No newline at end of file

Add trailing newline

(W292)